tests_config/run_spec.js

```diff
@@ -18,17 +18,25 @@ const CURSOR_PLACEHOLDER = '<|>';
 const RANGE_START_PLACEHOLDER = '<<<PRETTIER_RANGE_START>>>';
 const RANGE_END_PLACEHOLDER = '<<<PRETTIER_RANGE_END>>>';
 
-// TODO: these test files need fix
+// Here we add files that will not be the same when formating a second time.
 const unstableTests = new Map(
-  ['Assembly/Assembly.sol'].map((fixture) => {
+  [].map((fixture) => {
     const [file, isUnstable = () => true] = Array.isArray(fixture)
       ? fixture
       : [fixture];
     return [path.join(__dirname, '../tests/', file), isUnstable];
   })
 );
 
-const unstableAstTests = new Map();
+// Here we add files that will not have the same AST after being formated.
+const unstableAstTests = new Map(
+  [].map((fixture) => {
```

Comment:
same as above

Reply:
I decided to add the function within since an empty `new Map();` doesn't really explain how this map should be implemented in case we had an example where ASTs are changed. 
We don't have inconsistent AST but we do sometimes change it in the parsing step.
Stuff like changing `uint` to `uint256` and enforcing parentheses for easy to read code. For more correctness we could also add a new test that compares the compiled code of contracts where we know that the AST changes.
Until now I have used this [project](https://github.com/Janther/prettier-ast-changing-proposals) to prove that those AST changes don't affect the compiled code, but would be a nice addition to our process and we could keep track on new compiler versions that might affect this.